spack.yaml

```diff
@@ -4,58 +4,61 @@
 # configuration settings.
 spack:
   specs:
-    # TODO: Replace the MODEL and VERSION.
-    # The root SBD for the model and overall version of the deployment:
-    # - MODEL@git.VERSION
+    - access-am3@git.2025.06.000
   packages:
-    # TODO: Specify versions and variants of dependencies where required
-    # Specification of dependency versions and variants goes here.
-    # CI/CD requires that the first element of the require is only a version:
-    # openmpi:
-    #   require:
-    #     - '@4.0.2'
+    # Direct dependencies
+    um:
+      require:
+        - '@13.1'
+        - model="vn13p1-am"
+
+    # Indirect dependencies
+    eccodes:
+      require:
+         - '@2.34.0'
```

Comment:
As far as I can tell we're not building with this variant. If so we should remove this.